src/Components/SvaFinder.js

```diff
@@ -140,18 +141,20 @@ const steps = {
     },
 };
 
-const initial_state = {
-    step: steps['country'],
-    prev_state: null,
-    question: t('country', 'sva-finder'),
-    result: false,
-};
-
 export default class SvaFinder extends Component {
     constructor(props) {
         super(props);
 
-        this.state = initial_state;
+        this.steps = window?.props?.override ? deepmerge(general_steps, window.props.override) : general_steps;
+
+        this.initial_state = {
```

Comment:
Why move these to instance properties?

Reply:
I planned to pass the override to the react component. You are right, we don't need this anymore.